src/test/Directory.Build.targets

```diff
@@ -84,6 +84,7 @@
       <TestContextVariable Include="MNA_TFM=$(NETCoreAppFramework)" />
       <TestContextVariable Include="DOTNET_SDK_PATH=$(DotNetRoot)" />
       <TestContextVariable Include="TEST_PROJECTS_PROPS_TARGETS_FOLDER=$(MSBuildThisFileDirectory)" />
```

Comment:
I think it would actually be a good idea to move `TestProjects.props/targets` into the `Assets/TestProjects` dir and remove `TEST_PROJECTS_PROPS_TARGETS_FOLDER` (usage now covered by `TEST_ASSETS`). They really are test assets, they were just in a weird place before because they were discovered by dir traversal.

Not a necessary change, but would be nice if you have time. 🙂

Reply:
I'll add that to this PR.